app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -43,6 +43,15 @@ def new
       @need_to_set_up_additional_mfa = need_to_set_up_additional_mfa?
 
       if result.errors.present?
+        if @platform_authenticator
+          increment_mfa_selection_attempt_count(
+            TwoFactorAuthenticatable::AuthMethod::WEBAUTHN_PLATFORM,
+          )
+        else
+          increment_mfa_selection_attempt_count(
+            TwoFactorAuthenticatable::AuthMethod::WEBAUTHN,
+          )
+        end
```

Comment:
We can use your new helper method to simplify

```suggestion
        increment_mfa_selection_attempt_count(webauthn_auth_method)
```

Reply:
oh right! thanks :)